torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -99,6 +105,9 @@ def _forward(self, input):
             zero_bias = torch.zeros_like(self.bias)
         else:
             zero_bias = torch.zeros(self.out_channels, device=scaled_weight.device)
+        # mypy error: "Tensor" not callable  [operator]
+        # self._conv_forward is a method
+        assert not isinstance(self._conv_forward, torch.Tensor)
```

Comment:
Weird this is needed, Mypy cannot make sense of the inherited `nni._FusedModule` it looks like. Also fine.

Reply:
Hi @guilhermeleobas, I remember there were some mypy caching issues previously. did you try deleting the mypy cache and rerun test/test_type_hints.py without this assert?